src/app/games/lasca/LascaTutorial.ts

```diff
@@ -0,0 +1,95 @@
+import { Tutorial, TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { Coord } from 'src/app/jscaip/Coord';
+import { LascaMove } from './LascaMove';
+import { LascaPiece, LascaSpace, LascaState } from './LascaState';
+
+const zero: LascaPiece = LascaPiece.ZERO;
+const one: LascaPiece = LascaPiece.ONE;
+const _u: LascaSpace = new LascaSpace([zero]);
+const _v: LascaSpace = new LascaSpace([one]);
+const uv: LascaSpace = new LascaSpace([zero, one]);
+const __: LascaSpace = LascaSpace.EMPTY;
+
+export class LascaTutorial extends Tutorial {
+    public tutorial: TutorialStep[] = [
+        TutorialStep.informational(
+            $localize`Lasca: origins`,
+            $localize`Lasca is a game based on draughts created in 1911 by Emanuel Lasker, chess world champion. It's played on a 7x7 board, each player has 11 pieces.`,
+            LascaState.getInitialState(),
+        ),
+        TutorialStep.informational(
+            $localize`Goal of the game`,
+            $localize`The goal of Lasca is, like for draughts, to render the opponent out of possible move, either by capturing all his pieces, either by blocking them.`,
```

Comment:
"to render the opponent unable to move"

T'as-tu pas d'outil de recherche dans le code pour avoir pas retrouvé ceci ? :D

Reply:
de ce que je voit ici je l'ai jamais modifié :D ! Réglé monsieur passif aggressif !

Reply:
je t'avais fait le commentaire dans les traductions et tu trouvais pas le code en question !
C'est du code lasca, comment ça tu l'as jamais modifié ? Tu l'as ÉCRIS :D

Reply:
Oui mais je l'avais pas encore fait passer à "to render", bref, on est bon :D ?